addons/website/views/snippets/s_tabs.xml

```diff
@@ -20,13 +20,19 @@
                 </div>
                 <div class="s_tabs_content tab-content">
                     <div class="tab-pane fade show active" id="nav_tabs_content_1" role="tabpanel" aria-labelledby="nav_tabs_link_1">
-                        <div class="oe_structure oe_empty"/>
+                        <div class="oe_structure">
+                            <t t-snippet-call="website.s_text_block"/>
+                        </div>
                     </div>
                     <div class="tab-pane fade" id="nav_tabs_content_2" role="tabpanel" aria-labelledby="nav_tabs_link_2">
-                        <div class="oe_structure oe_empty"/>
+                        <div class="oe_structure">
+                            <t t-snippet-call="website.s_text_block"/>
+                        </div>
                     </div>
                     <div class="tab-pane fade" id="nav_tabs_content_3" role="tabpanel" aria-labelledby="nav_tabs_link_3">
-                        <div class="oe_structure oe_empty"/>
+                        <div class="oe_structure">
+                            <t t-snippet-call="website.s_text_block"/>
+                        </div>
```

Comment:
Should not you keep the `oe_empty` class on the `oe_structure` here? This is not a class to say that the element is empty but to say there should be the "drag and drop here" message when the `oe_structure` is empty.